src/Spryker/Console/InstallConsoleCommand.php

```diff
@@ -236,6 +237,30 @@ protected function getEnvironment()
     {
         $environment = (getenv('APPLICATION_ENV') ? getenv('APPLICATION_ENV') : 'development');
 
-        return $environment;
+        return (string)$environment;
+    }
+
+    /**
+     * @param string $optionName
+     *
+     * @throws \Spryker\Install\Exception\InstallException
+     *
+     * @return string
+     */
+    protected function getOptionAsString(string $optionName): string
+    {
+        $recipeOptionValue = $this->input->getOption($optionName);
+
+        if (!is_string($recipeOptionValue)) {
+            throw new InstallException(
+                sprintf(
+                    'Value of `%s` option should return `STRING` type. Return `%s`.',
```

Comment:
string

Reply:
The method was deleted because we have an option in one place and argument in other with different return types